src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: $width-of-items;
```

Comment:
It seems that the variable `$width-of-items` is not defined in this file. Make sure that you have imported the file where it is defined.